glyphs-reader/src/font.rs

```diff
@@ -25,12 +25,24 @@ const V3_METRIC_NAMES: [&str; 5] = [
 ];
 const V2_METRIC_NAMES: [&str; 5] = ["ascender", "baseline", "descender", "capHeight", "xHeight"];
 
-#[derive(Debug, FromPlist, PartialEq, Eq, Hash)]
+#[derive(Debug, PartialEq, Eq, Hash)]
 pub struct Font {
+    pub family_name: String,
+    pub axes: Vec<Axis>,
+    pub font_master: Vec<FontMaster>,
+    pub default_master_idx: usize,
+    pub glyphs: BTreeMap<String, Glyph>,
+    pub glyph_order: Vec<String>,
+    pub codepoints: BTreeMap<String, Vec<u32>>,
```

Comment:
I strongly object to having a top-level `codepoints` mapping glyph names to unicode codepoints (u32), for two reasons:

1) cmap table (which is what this is about) maps codepoints to glyph (indices, or at higher level, glyph names), not the other way around. Thinking like a glyphs has this and that codepoint is the wrong way to think about them, as it may involuntarily lead to multiple glyphs being mapped to the same codepoint, which does not make any sense in a font. Only one glyph can be mapped to a given codepoint inside cmap table (while multiple codepoints may well map to the same glyph). In UFO4 the plan was/is to get rid of uncode elements inside GLIFs and add a cmap.txt csv file at the global level that maps unicodes to glyph names.
2) Moving this (inverted) cmappings to the Font level does not add/improve anything to keeping them inside the Glyph structure and doing the usual font["glyphname"].unicodes, which is what our sources already contain.

So if you like to have this top-level map (i'm not sure we need that here because (2) above), I recommed you invert the keys and values, and make it a map from u32 to String. Maybe we just do that for the IR and here at the source level we stay closer to the sources (where glyph "have" codepoints).

Reply:
For context, I want IR to separate the information to build cmap so it can be it's own task and not load glyphs. However, your point about the direction being wrong - even if it does mirror how Glyphs appears to think about it - stands so i will fix the direction.

Reply:
Fixed.